dev/test-in-rancher

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -euxo
+
+branch=${CHARTS_BRANCH-""}
+
+# Build Rancher
+# This is not needed if all you want is to test Fleet against an existing Rancher release
+#pushd ../rancher
+#make quick
+#popd
+
+# XXX: this could be improved by using a single server instead of 3; calling this script is lazy but overkill
+./dev/setup-k3d
```

Comment:
I'm in favor of deleting that script. Just copy what you need here? Or remove the setup, creating k3d or re-using one can be left to the dev. If this script creates the k3d, does it also have to remove it?

Reply:
Good point, I've removed that call and documented cluster availability as a prerequisite to running this script.